### ISSUE TITLE
tutorial/congratulations seems redundant on learn.svelte.dev

### ISSUE THREAD

**Issue by lxhom** [sveltejs/learn.svelte.dev#212]
From https://learn.svelte.dev/tutorial/congratulations:

> You've now finished the Svelte tutorial and are ready to start building apps. You can refer back to individual chapters at any time (click the title above to reveal a dropdown) or continue your learning via the [API reference](https://svelte.dev/docs), [Examples](https://svelte.dev/examples) and [Blog](https://svelte.dev/blog). If you're a Twitter user, you can get updates via [@sveltejs](https://twitter.com/sveltejs).

This is right before the advanced SvelteKit chapter, which is a bit out of place. I'd either edit it to be Svelte-specific, delete it or move it behind advanced SvelteKit. (I did not do this as a PR because there are multiple ways to solve it, but I'd be down to write something new for it!)

**Comment by Rich-Harris** [comment]
agree! am collecting similar examples in #281 so they're all in one place — closing this in favour of that issue
